Postgres/MySQL/MSSQL: Adds support for region annotations (#20752)
grafana/grafana: docs/sources/features/datasources/postgres.md

Name | Description
------------ | -------------
time | The name of the date/time field. Could be a column with a native sql date/time data type or epoch value.

> timeend | Optional name of the time end field, needs to be date/time data type. If set, then annotations will be marked as a regions between time and time-end.

Reviewer: ```suggestion
timeend | Optional name of the time end field, needs to be date/time data type. If set, then annotations are marked as regions between time and time-end.
```
